### ISSUE TITLE
Python 3: brackets removed from print?

### ISSUE THREAD

**Issue by basbebe** [josecols/python-tidy#2]
python-tidy seems to remove the brackets from print commands.
Thats not so nice for use with Python 3…

**Comment by byteknacker** [comment]
I have the same problem. In python 2 this can also cause some break downs since some remote environments will reject our code. It is not stable without brackets.

**Comment by josecols** [comment]
Yes, I have confirmed this issue, but the problem is originated by the PythonTidy.py package used to format the code.
